lib/ansible/modules/cloud/cloudstack/cs_instance_info.py

```diff
@@ -43,17 +46,28 @@
 '''
 
 EXAMPLES = '''
-- name: gather instance facts
-  cs_instance_facts:
+- name: gather instance info
+  cs_instance_info:
     name: web-vm-1
   delegate_to: localhost
   register: vm
 
-- debug:
-    var: cloudstack_instance
-
-- debug:
+- name: Show the returned results of the registered variable
+  debug:
     var: vm
+
+# When the module is called as cs_instance_info, return values are also
+# published in ansible_facts['cloudstack_instance'] and can be used as
+# follows. Note that this is deprecated and will stop working in
+# Ansible 2.13.
+- name: gather instance info
+  cs_instance_facts:
```

Comment:
I'm confused here - line 59 says 'when the module is called as cs_instance_info..." but on this example, it's using cs_instance_facts:

Reply:
Thanks! I must have copy'n'pasted that wrongly... Fixed both.